### DISCUSSION TITLE
AdSense

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#505] [Q&A] [answered]
if we have AdSense active on our domain  so can we play ads on it?

**Comment by spencerwooo** [comment, marked as answer]
that would be for u to decide, you can do whatever you want on your side.

**Reply by hirockstar** [reply]
oh okey thanks

**Reply by hirockstar** [reply]
btw I translate to Hindi [common.json](https://rdrive.in/api/raw/?path=/index/hi/common.json) can you please update it.

**Comment by hirockstar** [comment]
got solution for adsense 
just add this code in path:- pages/_document.tsx

in head tag
[here](https://github.com/vercel/next.js/discussions/18801#discussioncomment-121458)
